### DISCUSSION TITLE
Welcome to rpivotTable Discussions!

### DISCUSSION THREAD

**Discussion by fraupflaume** [fraupflaume/rpivotTable#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ghost** [comment]
I can't possibly thank you enough for the help you have offered about this widget. Regarding your famous  application for configuration Storage / Recovery, I would suggest you maintain the latest version of your code in a repository, like [this](https://gist.github.com/fraupflaume/). In the relevant SO answers you could also include the corresponding link. I am afraid that, if parts of your code remain only in scattered questions, we will all be lost.
&mdash; Best regards

**Reply by fraupflaume** [reply]
That's a good idea. There's quite a bit out there now.
